### PR TITLE
Restore AI correlation changes in public doc (Publication date TBD)

### PR DESCRIPTION
**Pull request by rhetoric101**
Since we pulled back the original NRAI correlation changes in PR #7511, this is a PR to publish those original changes when the product is ready.

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 1]
```diff
@@ -144,17 +157,19 @@ When building a decision, steps 1, 2 and 3 are optional on their own but at leas
 
     <tr>
       <td>
-        Step 3: Topology correlation
+        Step 3: Correlate topology (optional)
       </td>
 
       <td>
-        [Topology correlation](#topology) is currently in limited release. Contact your account representative to enable this feature. Once enabled, you'll first need to ingest your topology data via our NerdGraph `aiTopologyCollector` (search for aiTopology in the [NerdGraph GraphiQL explorer](https://api.newrelic.com/graphiql)). This allows any topology-related decision to be matched with your topology data. [Learn more about setting up topology correlation](#topology).
+        For automatic topology correlation, make sure your telemetry data is collected by [New Relic agents](/docs/new-relic-solutions/new-relic-one/install-configure/compatibility-requirements-new-relic-agents-products/). Learn more about [topology correlation out-of-the-box]([#topology-requirements]).
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:164]: The "topology-requirements" link gives an error. I think we need to remove the [] inside the ().

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 2]
```diff
@@ -870,93 +890,90 @@ By default, the attributes are sorted by frequency with the least frequently rep
...
+* **Edges:** An edge is a connection between two vertices. Edges describe the relationship between vertices.
 
-To set up your topology, you'll use [our NerdGraph API](#create-topology-graph). But first, it may help to understand how topology is used to correlate incidents:
+It may help to understand how topology is used to correlate incidents:
 
-1. First, New Relic gathers all relevant incidents. This includes incidents where [decision-logic steps 1 and 2](#build-decision) are true, and that are also within the defined time window in advanced settings.
+1. First, New Relic gathers all relevant incidents. This includes incidents where [decision logic steps 1 and 2](#build-decision) are true and that are also within the defined time window in advanced settings.
+
+  In this example, all of the incidents in the dotted-line selection have met these requirements: they've gone through the [decision logic in steps 1 and 2](#build-decision), and all contextual comparisons made in [decision logic step 2](#build-decision) are true.
```
Comment (barbnewrelic) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:942]: Not sure where the anchors in this step are supposed to go, but they don't work in Gatsby preview. Are they really needed?

**Review by barbnewrelic — APPROVED** [review]
@rhetoric101  A couple of minor things re: anchors in this doc. If you decide to fix them, go ahead and merge when your SME approves. No need for TW review again. Thanks again for taking care of my original blooper that caused you to have to recreate this PR!

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:164]: @barbnewrelic, good catch! I just fixed this!

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 2 above)
Comment (rhetoric101) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:942]: @barbnewrelic, thanks for catching this oversight! I think these were all supposed to point to the "custom" anchor, but I will confirm.

**Pull request merged**